src/john.c

```diff
@@ -1956,14 +1963,11 @@ static void john_done(void)
 		} else if (children_ok) {
 			log_event("Session completed");
 			if (john_main_process) {
-				fprintf(stderr, "Session completed. %s\n", mode_exit_message);
+				fprintf(stderr, "Session%s completed. %s\n", john_session_name, mode_exit_message);
```

Comment:
Let's drop the trailing dot here while we're at it. It's been annoying me all the time. ;-)